### PR TITLE
Traitor kill announcer

### PR DESCRIPTION
**Pull request by SECBATON-GRIFFON**
[content][gamemode]
<video src="https://github.com/user-attachments/assets/616fbc88-170e-4f2f-833d-7a6c21ebc477">

## What this does
adds an item called the KILLbot to the pointless badassery section of uplinks for 1 telecrystal, that announces various relevant lines from unreal tournament as seen above

## Why it's good
murderbone flavour

## How it was tested
see video

## Changelog
:cl:
 * rscadd: Uplinks now have a new item in the pointless badassery section: KILLbots! These sound systems make sounds for various events such as killstreaks, headshots and other miscellaneous round related things when held by someone.

### CODE REVIEW

**Comment by Eneocho** [comment]
surely this won't cause issues

**Comment by MercuriusGH** [comment]
Can you hear this on the entire Z level?

**Comment by SECBATON-GRIFFON** [comment]
> Can you hear this on the entire Z level?

just around the person using it as normal

**Comment by MercuriusGH** [comment]
> > Can you hear this on the entire Z level?
> 
> just around the person using it as normal

WEAK
ideally it should display text in the middle of your screen (with a portrait made from the head of the fragger)

**Comment by SECBATON-GRIFFON** [comment]
> > > Can you hear this on the entire Z level?
> > 
> > 
> > just around the person using it as normal
> 
> WEAK ideally it should display text in the middle of your screen (with a portrait made from the head of the fragger)

alright fine, i'll make this an option for when this thing gets arcane tampered, maybe

**Comment by DrSnips** [comment]
>arcane tamper on a traitor item in the pointless badass category.

Don't even bother at that point. That's some never going to happen shit.

**Comment by Eneocho** [comment]
make it very obvious like mercurius suggests, that's more fun

**Comment by ErrorsWindows** [comment]
can you do the execution machete that hippie station has, where you go to behead someone and it announces it to the entire station as a progress bar fills for 30 seconds while an osama bin laden speech plays in the background across the entire z level

File: code/game/objects/items/devices/roganbot.dm [hunk 1]
```diff
@@ -226,3 +226,215 @@ var/global/list/number2rogansound = list() //populated by /proc/make_datum_refer
...
+							failed_jectie = TRUE
+							break
+					if(failed_jectie)
+						break
+			if(flawless_victory)
+				playsound(loc,'sound/effects/2003M/Flawless_victory.ogg',100)
+			else if(teamwon)
+				playsound(loc,'sound/effects/2003M/You_have_won_the_match.ogg',100)
+			else
+				playsound(loc,'sound/effects/2003M/You_have_lost_the_match.ogg',100)*/
```
Comment (west3436) [inline code/game/objects/items/devices/roganbot.dm:440]: ?

File: data/bunker.txt
Comment (west3436) [inline data/bunker.txt:1]: Remove this one; it's in .gitignore now

**Review by west3436 — CHANGES_REQUESTED** [review]

File: data/bunker.txt
Comment (SECBATON-GRIFFON) [reply data/bunker.txt:1]: aye, it was getting annoying

File: code/game/objects/items/devices/roganbot.dm (hunk 1 above)
Comment (SECBATON-GRIFFON) [reply code/game/objects/items/devices/roganbot.dm:440]: code that doesn't work that i left in and commented to see if i could make it work, could just remove it entirely if you want

File: data/bunker.txt
Comment (SECBATON-GRIFFON) [reply data/bunker.txt:1]: aye, it was getting annoying. done

**Pull request merged**